cmgd/cmgd_db.h

```diff
@@ -173,5 +185,7 @@ extern void cmgd_db_status_write_one(
         struct vty *vty, cmgd_db_hndl_t db_hndl);
 
 extern void cmgd_db_status_write(struct vty *vty);
-
+extern int cmgd_db_dump_in_memory(
```

Comment:
Again of you move the commit-info stuff to cmgd_db.c you won't need to expose this.